src/ophys_etl/modules/segmentation/modules/README.rst

```diff
@@ -0,0 +1,38 @@
+Segmentation Steps
+==================
+- Detect (feature_vector_segmentation.py)
+- Merge (roi_merging.py)
```

Comment:
Do you mind putting an optional filter step between Detect and Merge? I imagine we may want to do some filtering there to prevent the merge module from getting distracted by obvious chaff.

Reply:
done.